src/main/java/edu/northeastern/truthtree/service/basicInfo/BasicInfoService.java

```diff
@@ -0,0 +1,30 @@
+package edu.northeastern.truthtree.service.basicInfo;
+
+import org.json.simple.JSONArray;
+
+import edu.northeastern.truthtree.adapter.basicInfo.BasicInfoDBAdapter;
+import edu.northeastern.truthtree.adapter.basicInfo.BasicInfoMockAdapter;
+import edu.northeastern.truthtree.adapter.basicInfo.IBasicInfoAdapter;
+import edu.northeastern.truthtree.adapter.collections.CollectionsMockAdapter;
+import edu.northeastern.truthtree.adapter.collections.ICollectionAdapter;
+
+public class BasicInfoService implements IBasicInfoService {
+
+	@Override
+	public JSONArray getBasicStatesInfo() {
+		IBasicInfoAdapter adapter = new BasicInfoMockAdapter();
```

Comment:
Consider using beans to configure the adapter. You can refer to AttributesService